boards/arm/beagle_bcf/beagleconnect_freedom.dts

```diff
@@ -56,15 +56,6 @@
 				<&gpio0 30 GPIO_ACTIVE_HIGH>; // SubG TX/RX 0dB
 		};
 	};
-
-	sens_i2c: sensor-switch {
```

Comment:
OK, it looks like the way I had it before made more sense. I just listed a controller to define this new bus. I think @Ayush1325 might have had a different perspective on it, but I think the previous instance method was fine.